### PR TITLE
Handle window resize

### PR DESCRIPTION
**Pull request by ashutoshtanwar**

### CODE REVIEW

File: lib/SwipeableWrapper/index.jsx [hunk 1]
```diff
@@ -61,7 +59,7 @@ const SwipeableWrapper = forwardRef(
         rAF(() => {
           onSlideChange(index.current);
           scrollToTop();
-          const height = innerHeight - elementRef.current.clientTop;
+          const height = window.innerHeight - elementRef.current.clientTop;
```
Comment (Gitesh-Narula) [inline lib/SwipeableWrapper/index.jsx:62]: What happens on SSR where `window` object is not defined?

Comment (Gitesh-Narula) [reply lib/SwipeableWrapper/index.jsx:62]: Also, use optional chaining for elementRef.

**Review by Gitesh-Narula — COMMENTED** [review]
Added Comments.

File: lib/SwipeableWrapper/index.jsx (hunk 1 above)
Comment (ashutoshtanwar) [reply lib/SwipeableWrapper/index.jsx:62]: > What happens on SSR where `window` object is not defined?

We are using `window` object only inside functions that are used inside `useLayoutEffect` which will not be called on SSR.

**Review by Gitesh-Narula — APPROVED** [review]

**Pull request merged**